pkg/stringid/stringid.go

```diff
@@ -47,3 +51,29 @@ func GenerateRandomID() string {
 		return value
 	}
 }
+
+// GenerateNonCryptoID generates unique id without using cryptographically
+// secure sources of random.
+// It helps you to save entropy.
+func GenerateNonCryptoID() string {
+	r := mrand.New(random.Source)
+	id := make([]byte, 32)
+	for {
+		if _, err := io.ReadFull(rand.Reader, id); err != nil {
+			panic(err) // This shouldn't happen
+		}
+		binary.PutUvarint(id, uint64(r.Int63()))
+		binary.PutUvarint(id[8:], uint64(r.Int63()))
+		binary.PutUvarint(id[16:], uint64(r.Int63()))
+		// because PutUvarint writing 9 bytes
+		binary.PutUvarint(id[23:], uint64(r.Int63()))
+		value := hex.EncodeToString(id)
```

Comment:
starting from here, it should be factored out in a function reused by both GenerateNonCryptoID and GenerateRandomID